### DISCUSSION TITLE
Uninstall instructions/script

### DISCUSSION THREAD

**Discussion by kflak** [lumen-oss/rocks.nvim#328] [Ideas]
[Issue #312](https://github.com/nvim-neorocks/rocks.nvim/issues/312) made me realize I don't know how to cleanly get rid of rocks on my system, in case I would need to troubleshoot a botched install. Would be nice to have some instructions, or, even better, a cleanup script that could remove all traces of rocks. _Even_ better would be a script that backs up the current config, so that it could be restored afterwards :-)

**Comment by mrcjkb** [comment]
Related: https://github.com/nvim-neorocks/rocks.nvim/issues/260

**Reply by kflak** [reply]
Right! And I suppose with this it would be trivially easy to include a simple `:UninstallRocks` command as well?

**Reply by mrcjkb** [reply]
Something like that. I'll have to put some thought into it, because we don't want to make it easy to accidentally uninstall rocks.nvim.

**Reply by kflak** [reply]
You could map it to ctrl-alt-delete 😁
